target/linux/ramips/image/rt305x.mk

```diff
@@ -296,6 +296,7 @@ TARGET_DEVICES += dir-620-d1
 
 define Device/dwr-512-b
   DTS := DWR-512-B
+  IMAGE_SIZE := 7700k
   DEVICE_TITLE := D-Link DWR-512 B
```

Comment:
this is an unrelated change. either update the description of the patch or remove this chunk